### ISSUE TITLE
Joining a team doesn't reset the Select a Project Tab until page is refreshed

### ISSUE THREAD

**Issue by MaxBittker** [getsentry/sentry#5123]

**Comment by MaxBittker** [comment]
this issue actually still exists in a less serious form: you have to toggle the dropdown twice to see the new values, it should be fixed.
additionally, i think that the dropdown should say "join a team" when its empty, but that's its own issue

**Comment by gricey432** [comment]
Came here to report this on hosted sentry. I've joined a team and clicked around in that team a bit but it's still not in my project selector.
It eventually appeared only after a page refresh.

**Comment by BYK** [comment]
Closing this issue due to staleness. Feel free to comment here if you think we should still work on this.

**Comment by gricey432** [comment]
Just tried it out on hosted sentry and this appears to be fixed anyway
